pkg/kudoctl/kudoinit/types.go

```diff
@@ -13,7 +13,16 @@ const (
 )
 
 type InitStep interface {
+	// Returns a one-line description to print
+	Description() string
```

Comment:
Consider embedding the Stringer interface instead: https://godoc.org/fmt#Stringer - this would allow this to work via fmt packages rather than having to create all of that code yourself!

Reply:
Done